src/main/java/com/joss/conductor/mobile/ConductorConfig.java

```diff
@@ -385,6 +386,10 @@ public void setScreenshotOnFail(boolean screenshotOnFail) {
         this.screenshotOnFail = screenshotOnFail;
     }
 
+    public boolean isScreenshotOnSkip() { return screenshotOnSkip; }
```

Comment:
to use consistent style let's put the statements on it's own line. Like in the other methods.